docs/source/community.rst

```diff
@@ -0,0 +1,69 @@
+Community
+=========
+
+Intake is used and developed by individuals at a variety of institutions.  It
+sits within the broader Python numeric ecosystem commonly referred to as PyData
+or SciPy.
+
+Discussion
+----------
+
+Conversation happens in the following places:
+
+1.  **Usage questions** are directed to `Stack Overflow with the #intake tag`_.
+    Intake developers monitor this tag.
+2.  **Bug reports and feature requests** are managed on the `GitHub issue
+    tracker`_
```

Comment:
I would also mention the set of links on the drivers page, in case the issue would better go to one of the related repos